common/genes/01_genes_morph.txt

```diff
@@ -340,6 +340,89 @@ age_presets = {
 }
 
 morph_genes = {
+	# Warcraft
+	gene_race = {
+		index = 112
```

Comment:
I assume you meant to make the right collumn aligned, however it isn't.

Reply:
> I assume you meant to make the right collumn aligned, however it isn't.

Didn't get what is the right column? `index = 112`? It's aligned, it has 2 tabs, the `gene_race = {` has 1 tab.

Reply:
I meant for the following lines. They are not aligned. 

![image](https://user-images.githubusercontent.com/4825809/122440704-1db2bc00-cf9d-11eb-9c44-dc60ea92c246.png)